code/modules/crafting/craft.dm

```diff
@@ -431,3 +435,19 @@
 	data["tool_text"] = tool_text
 
 	return data
+
+//Mob helpers
+
+/mob/proc/known_recipes(datum/crafting_recipe/R)
+	if(!mind)
+		return FALSE
+	if(!mind.learned_recipes)
+		return FALSE
```

Comment:
can't this just be rewritten as 
```DM
if(mind?.learned_recipes)
    return (R.type in mind.learned_recipes)
return FALSE
```

Reply:
if you are golfing this, `return R.type in mind?.learned_recipes` because `_ in null` is always false

Reply:
Opted with moving these to mind to keep this consistent with spells.